deploy/Dockerfile_debug

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.10
+ADD https://github.com/GoogleCloudPlatform/docker-credential-gcr/releases/download/v1.4.3-static/docker-credential-gcr_linux_amd64-1.4.3.tar.gz /usr/local/bin/
+RUN tar -C /usr/local/bin/ -xvzf /usr/local/bin/docker-credential-gcr_linux_amd64-1.4.3.tar.gz
+
+FROM gcr.io/distroless/base:latest
```

Comment:
why not a `base:debug` that already contains a busybox shell?

Reply:
good point.
I was not aware of the `base:debug` image. I just looked at Kaniko ...

I will update the PR accordingly